samples/Nancy.Demo.Hosting.Kestrel/DemoBootstrapper.cs

```diff
@@ -0,0 +1,23 @@
+namespace Nancy.Demo.Hosting.Kestrel
+{
+    using System;
+    
+    public class DemoBootstrapper : DefaultNancyBootstrapper
+    {
+        public DemoBootstrapper()
+        {
+            
```

Comment:
This is not the blank line you were looking for, please try again.. `hint it should be two rows down` :smile: